Detectors/ITSMFT/ITS/tracking/src/TimeFrame.cxx

```diff
@@ -275,6 +278,27 @@ void TimeFrame::initialise(const int iteration, const MemoryParameters& memParam
   mRoads.clear();
   mRoadLabels.clear();
 
+  mMSangles.resize(trkParam.NLayers);
+  mPhiCuts.resize(mClusters.size() - 1, 0.f);
+
+  float oneOverR{0.001f * 0.3f * std::abs(mBz) / trkParam.TrackletMinPt};
+  for (unsigned int iLayer{0}; iLayer < mClusters.size(); ++iLayer) {
+    mMSangles[iLayer] = MSangle(0.14f, trkParam.TrackletMinPt, trkParam.LayerxX0[iLayer]);
+    mPositionResolution[iLayer] = std::hypot(trkParam.LayerMisalignment[iLayer], trkParam.LayerResolution[iLayer]);
+
+    if (iLayer < mClusters.size() - 1) {
+      const float& r1 = trkParam.LayerRadii[iLayer];
+      const float& r2 = trkParam.LayerRadii[iLayer + 1];
+      const float res1 = std::hypot(trkParam.PVres, mPositionResolution[iLayer]);
+      const float res2 = std::hypot(trkParam.PVres, mPositionResolution[iLayer + 1]);
+      const float cosTheta1half = std::sqrt(1.f - Sq(0.5f * r1 * oneOverR));
+      const float cosTheta2half = std::sqrt(1.f - Sq(0.5f * r2 * oneOverR));
+      float x = r2 * cosTheta1half - r1 * cosTheta2half;
+      float delta = std::sqrt(1. / (1.f - 0.25f * Sq(x * oneOverR)) * (Sq(0.25f * r1 * r2 * Sq(oneOverR) / cosTheta2half + cosTheta1half) * Sq(res1) + Sq(0.25f * r1 * r2 * Sq(oneOverR) / cosTheta1half + cosTheta2half) * Sq(res2)));
```

Comment:
better to put `1.f` everywhere (`1.f/(...`)! Otherwise, the division will be done as double.

Reply:
Indeed! Sometimes, on some machine I even get the warning to fix this. In the next PR it will be done